deck/migrations/0018_event_paper_closing_date.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```

Comment:
If you update your branch with the Master, you will not need to implement this, there was a pull request #321 that refactor this and standarlize to `closing_date`.